src/librustdoc/html/render/context.rs

```diff
@@ -43,6 +43,8 @@ use crate::try_err;
 /// It is intended that this context is a lightweight object which can be fairly
 /// easily cloned because it is cloned per work-job (about once per item in the
 /// rustdoc tree).
+
```

Comment:
Please remove this empty line (it looks super weird to have a doc comment not followed directly by the item).